demosplan/DemosPlanCoreBundle/Logic/Grouping/AbstractEntityGroup.php

```diff
@@ -16,6 +16,7 @@
 
 /**
  * @template T of \demosplan\DemosPlanCoreBundle\Entity\CoreEntity
+ *
```

Comment:
For me those 2 lines belong together and should not be separated by an empty line

Reply:
As above: I tried before to understand the docblock configuration options of csfixer and ended up in a corner questioning my life choices.